src/canvas/heatmapFeature.js

```diff
@@ -0,0 +1,207 @@
+var inherit = require('../inherit');
+var registerFeature = require('../registry').registerFeature;
+var heatmapFeature = require('../heatmapFeature');
+
+//////////////////////////////////////////////////////////////////////////////
+/**
+ * Create a new instance of class heatmapFeature
+ * Inspired from
+ *    https://github.com/mourner/simpleheat/blob/gh-pages/simpleheat.js
+ *
+ * @class
```

Comment:
This should be `@class geo.canvas.heatmapFeature` so it is labelled correctly in the API docs.